tests/model/test_compartmental.py

```diff
@@ -1,7 +1,9 @@
 import numpy as np
+
 from scipy.stats import gamma
 
-from cascade.model import (
+from cascade.stats import (
+    DemographicInterval,
```

Comment:
above in stats/__init__.py we have:
from cascade.stats.demography import DemographicInterval

Do we have that, so that here we don't have to have that?  Also, same goes for test_demography.py?

Reply:
Yeah, there is a convention in python that a sub-package pulls important names from it's modules into the sub-packages root level for easy importing and to make the public API clear. I tend not to do that, more from laziness than rational choice, so a lot of our code isn't written that way. It's happening in cascade.stats because of Drew.